internal/pkg/api/handleEnroll.go

```diff
@@ -211,6 +211,19 @@ func (et *EnrollerT) _enroll(
 	return &resp, nil
 }
 
+func removeDuplicateStr(strSlice []string) []string {
+	allKeys := make(map[string]bool)
```

Comment:
Nit: `map[string]struct{}` will use less memory than `map[string]bool`.

https://dave.cheney.net/2014/03/25/the-empty-struct

Reply:
Thanks for the link! I'm not sure about the readability vs. memory gain tradeoff. @michel-laterman any preference?

Reply:
I'm not sure there's a readability tradeoff.  Using `map[string]struct{}` is fairly common usage in Golang IME.  For example, there are plenty of occurrences in the Beats repo: https://github.com/search?q=repo%3Aelastic%2Fbeats%20map%5Bstring%5Dstruct%7B%7D&type=code

Reply:
Thanks, I'm very new to go so these comments are super helpful. I'll make the change with the implementation using Set from your other comment unless there are objections to it.